### PR TITLE
backend:Fix models user GetUserByName method

### PR DESCRIPTION
**Pull request by luolizhi**
**What type of PR is this?**
> Uncomment only one, leave it on its own line:
>
/kind bug
> /kind design
> /kind documentation
> /kind enhancement
> /kind feature

**What this PR does / why we need it**:

fix the code bug for models user GetUserByName.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

File: src/backend/models/user.go [hunk 1]
```diff
@@ -162,8 +162,8 @@ func (*userModel) GetUserById(id int64) (v *User, err error) {
 
 func (*userModel) GetUserByName(name string) (v *User, err error) {
 	v = &User{Name: name}
-	if err = Ormer().Read(v, "Name"); err == nil {
-		return v, nil
+	if err = Ormer().Read(v, "Name"); err != nil {
```
Comment (wilhelmguo) [inline src/backend/models/user.go]: 这么改有什么好处？

Comment (luolizhi) [reply src/backend/models/user.go]: 原来代码不会返回err，我认为是笔误。

Comment (luolizhi) [reply src/backend/models/user.go]: > 这么改有什么好处？

原来的代码不会返回err，只会返回nil。

Comment (wilhelmguo) [reply src/backend/models/user.go]: 确实是写错了，感谢纠正

**Pull request merged**